src/structures/paging/frame.rs

```diff
@@ -175,7 +175,7 @@ impl<S: PageSize> fmt::Debug for PhysFrameRange<S> {
 }
 
 /// An range of physical memory frames, inclusive the upper bound.
-#[derive(Clone, Copy, PartialEq, Eq)]
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
```

Comment:
Same point as above: I don't think that deriving `PartialOrd`/`Ord` makes sense here.